### ISSUE TITLE
Installation and code execution

### ISSUE THREAD

**Issue by MokeyDGarp** [gopro/OpenGoPro#38]
Hello, 

I am trying to control my two GoPro Hero9 via BLE by running a python script. Unfortunately I can't get any further with the documentation. I can neither connect the cameras nor execute commands. When I run the demo as in the documentation, I get the error message that 'GoPro()' does not exist. 

This is the code i used from the demo: 

![grafik](https://user-images.githubusercontent.com/89124934/129851205-fcf96ff8-96ad-425f-a0e7-2203164a2707.png)

I have installed the open-gopro with pip install open-gopro
What am i doing wrong?

Thank you very much for your help.

**Comment by MokeyDGarp** [comment]
I'm sorry, the error message appears when i run this code:

![grafik](https://user-images.githubusercontent.com/89124934/129853140-15d6ecab-efc8-4560-80d0-b5f035a1eaa8.png)

Thank you very much for your help.

**Comment by KonradIT** [comment]
Possible typo @tcamise-gpsw ? Should be called `GoPro` instead of `GoProBLE` in `.\demos\python\sdk_wireless_camera_control\docs\usage.rst`.

**Comment by MokeyDGarp** [comment]
If I use GoPro() instead of GoProBle() I get the error message from my first post.

**Comment by KonradIT** [comment]
Something weird with the system you're running if its not detecting any wifi interface.

**Comment by tcamise-gpsw** [comment]
Yeah, it is a typo in the documentation: it should be GoPro(). I'll fix these for the next release.

@MokeyDGarp What OS are you using? Also, can you attach the log file? It should be a "xxx.log" in the directory where you are running your test.

**Comment by MokeyDGarp** [comment]
Thank you. 

I am using Windows 10 Enterprise (64 Bit Version). 

It is the same error warning as in the other example. 

![bug gopro api](https://user-images.githubusercontent.com/89124934/130015892-ebaeea23-20d8-4837-8bf7-6f4c0e6e799d.PNG)

PLease find attached the .log file.

[photo.log](https://github.com/gopro/OpenGoPro/files/7012080/photo.log)

 Thank you very much for your help.

**Comment by tcamise-gpsw** [comment]
Well this is certainly an interesting problem. What's happening is that your system language is apparently German:
```
   MainThread:                open_gopro.util:   89 07:56:51.519 DEBUG    | Send cmd --> netsh wlan show interfaces
   MainThread:                open_gopro.util:   97 07:56:51.765 DEBUG    | Receive response --> Der Dienst fr die automatische WLAN-Konfiguration (wlansvc) wird nicht ausgefhrt.
```

The WiFi Controller parsers are looking for specific (English) strings which is why they are not able to parse the response.

Honestly, I do not have any good ideas of how to handle this. I will add this into the documentation that the code only supports English. 

The quickest workaround for you is probably to set your system language to English. 

As an actual solution, we could start adding support for other languages but this would likely need to come from someone who speaks the language (i.e. not me).
